Makefile

```diff
@@ -37,4 +37,34 @@ depends:
 	go get -d ./...
 
 build: depends
-	cd vflow; go build $(LDFLAGS) -o vflow $(GOFILES)
```

Comment:
The request about the make file.  Due to pushing all the logic into the make file we lose the build control of go tool thank.  When I removed GOFILES from here problems all go away :)